ci/test_windows.bat

```diff
@@ -0,0 +1,166 @@
+REM 
+REM Tests JDBC Driver on Windows
+REM
+setlocal
+setlocal EnableDelayedExpansion
+set TIMEOUT=90000
```

Comment:
let's name for what is this timeout

Reply:
The property was unused, so It was removed. Thanks